packages/Provider/src/Provider.tsx

```diff
@@ -1,6 +1,13 @@
 import * as React from "react";
 
-import { I18nProvider, type I18nProviderProps, useLocale, useLocalizedStringFormatter } from "react-aria";
+import { I18nProvider, 
+    type I18nProviderProps, 
+    useLocale, 
+    useLocalizedStringFormatter, 
+    useCalendar, 
```

Comment:
ok :(
I put them here because it was the only way I could get it to work with no errors, but I guess if you say to put the provider back, then it would probably work that way.